### PR TITLE
Fix Triage binary not running

### PR DESCRIPTION
**Pull request by michaelkolber**
ref: #18726

Add GOOS env variable to Triage build step

/cc BenTheElder spiffxp

### CODE REVIEW

File: triage/Dockerfile [hunk 1]
```diff
@@ -22,7 +22,7 @@ COPY . /test-infra/
 
 # Build Triage
 RUN cd /test-infra/triage/ && \
-    go build -v -o triage ./main.go
+    env GOOS=linux go build -v -o triage ./main.go
```
Comment (MushuEE) [inline triage/Dockerfile:25]: Do we still need the `apk add python2` below?

Comment (MushuEE) [reply triage/Dockerfile:25]: nvm!

Comment (michaelkolber) [reply triage/Dockerfile:25]: Yes, for the Cloud SDK

**Pull request merged**